### PR TITLE
Issue404 type

### PR DESCRIPTION
**Pull request by mwetter**
This will close https://github.com/iea-annex60/modelica-annex60/issues/404

All unit tests work.

### CODE REVIEW

File: Annex60/Fluid/BaseClasses/FlowModels/basicFlowFunction_dp.mo [hunk 1]
```diff
@@ -54,6 +54,18 @@ The input <code>m_flow_turbulent</code> determines the location of the regulariz
 </html>", revisions="<html>
 <ul>
 <li>
+January 22, 2016, by Michael Wetter:<br/>
+Corrected type declaration of pressure difference.
+This is
+for <a href=\"https://github.com/iea-annex60/modelica-annex60/issues/404\">#404</a>.
+</li>
+<li>
```
Comment (thorade) [inline Annex60/Fluid/BaseClasses/FlowModels/basicFlowFunction_dp.mo]: Revision pasted twice

File: Annex60/Fluid/BaseClasses/FlowModels/basicFlowFunction_dp_der2.mo [hunk 2]
```diff
@@ -40,6 +40,18 @@ with respect to the mass flow rate.
 revisions="<html>
 <ul>
 <li>
+January 22, 2016, by Michael Wetter:<br/>
+Corrected type declaration of pressure difference.
+This is
+for <a href=\"https://github.com/iea-annex60/modelica-annex60/issues/404\">#404</a>.
+</li>
+<li>
+January 22, 2016, by Michael Wetter:<br/>
```
Comment (thorade) [inline Annex60/Fluid/BaseClasses/FlowModels/basicFlowFunction_dp_der2.mo]: Revision pasted twice

File: Annex60/Fluid/BaseClasses/PartialResistance.mo [hunk 3]
```diff
@@ -82,6 +83,18 @@ this base class.
 </html>", revisions="<html>
 <ul>
 <li>
+January 22, 2016, by Michael Wetter:<br/>
```
Comment (thorade) [inline Annex60/Fluid/BaseClasses/PartialResistance.mo]: Revision pasted twice

File: Annex60/Fluid/Movers/BaseClasses/Characteristics/pressure.mo [hunk 4]
```diff
@@ -131,6 +133,18 @@ in all input variables.
 
 <ul>
 <li>
```
Comment (thorade) [inline Annex60/Fluid/Movers/BaseClasses/Characteristics/pressure.mo]: Revision pasted twice

File: Annex60/Fluid/Movers/BaseClasses/Characteristics/pressure.mo [hunk 5]
```diff
@@ -131,6 +133,18 @@ in all input variables.
...
+January 22, 2016, by Michael Wetter:<br/>
```
Comment (thorade) [inline Annex60/Fluid/Movers/BaseClasses/Characteristics/pressure.mo]: Revision pasted twice

**Comment by thorade** [comment]
Looks good to me, this should remove any ambiguity.
Some revisions are pasted twice.

**Comment by mwetter** [comment]
@thorade Thanks for catching the duplicate revision notes. I corrected it and will merge it to the master.

**Pull request merged**